src/index.html

```diff
@@ -5,8 +5,303 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="header__menu">
+        <ul class="menu">
+          <li class="menu__item "><a href="#" class="menu__link is-active">Apple</a></li>
```

Comment:
переноси длинные строки
```suggestion
          <li class="menu__item ">
            <a href="#" class="menu__link is-active">Apple</a>
          </li>
```